app/assets/stylesheets/sass/components/_pagination.scss

```diff
@@ -0,0 +1,47 @@
+.pagination {
+  $base-border-color: gainsboro !default;
+  $base-border-radius: 3px !default;
+  $base-spacing: 1.5em !default;
+  $pagination-border-color: $base-border-color;
+  $pagination-border: 1px solid $pagination-border-color;
+  $pagination-background: lighten($pagination-border-color, 10);
+  $pagination-hover-background: lighten($pagination-background, 5);
+  $pagination-color: $base-font-color;
+  text-align: center;
+
+  span.disabled {
+    display: none;
+  }
+
+  a {
+    display: inline;
+    margin: 0;
+    padding: 0;
+    text-align: center;
+    @include transition (all 0.2s ease-in-out);
+    background: $pagination-background;
+    border-radius: $base-border-radius;
+    border: $pagination-border;
+    color: $pagination-color;
+    outline: none;
+    padding: ($base-spacing / 4) ($gutter / 2);
+    text-decoration: none;
+
+    &:hover,
+    &:focus {
+      background: $pagination-hover-background;
+      color: $action-color;
+    }
+  }
+
+  em.current {
+    @include transition (all 0.2s ease-in-out);
+    background: $pagination-hover-background;
+    border-radius: $base-border-radius;
```

Comment:
Properties should be ordered background, border, border-radius, color, outline, padding, text-decoration